dnf-docker-test/features/steps/repo_steps.py

```diff
@@ -224,12 +224,13 @@ def given_repository_with_packages(ctx, rtype, repository):
         ctx.text = template.render(name=name, **settings)
         fname = "{!s}/{!s}.spec".format(tmpdir, name)
         step_a_file_filepath_with(ctx, fname)
+        buildname = '%{NAME}-%{VERSION}-%{RELEASE}.%{ARCH}.rpm'
```

Comment:
actually, Epoch is missing. and there are plans to add couple of tests for epoch... could you fix it?